Source/Tailwind.Traders.Web/ClientApp/src/pages/home/components/recommended/components/recommendedItem.js

```diff
@@ -1,15 +1,41 @@
 import React from "react";
 import { Link } from "react-router-dom";
 
-const RecommendedItem = ({ title, imageUrl, cssClass, url }) => {
+async function handleClickReward(cssClass, eventId) {
+    if (eventId) {
+        var rsp;
+        if (cssClass === "grid__item-a") {
```

Comment:
[nit] instead of making the entire `postReward` call conditional on the class it would cleaner to just make the reward conditional on the class.

```
var rewardValue = (cssClass === "grid__item-a") ? 1 : 0;
var rsp = await postReward(`/api/Personalizer/reward/${eventId}`, rewardValue);
```